src/main/java/com/agilethought/atsceapi/domain/UserDomain.java

```diff
@@ -0,0 +1,16 @@
+package com.agilethought.atsceapi.domain;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor 
```

Comment:
`@NoArgsConstructor` annotation is not necessary, unless you will define more than one constructor for this class. 
Java provides a default constructor that is noArgsConstructor.